src/transformers/configuration_bert.py

```diff
@@ -91,7 +91,11 @@ class BertConfig(PretrainedConfig):
             The epsilon used by the layer normalization layers.
         gradient_checkpointing (:obj:`bool`, `optional`, defaults to :obj:`False`):
             If True, use gradient checkpointing to save memory at the expense of slower backward pass.
-
+        position_embedding_config (:obj:`int`, `optional`, defaults to 0):
```

Comment:
can we call it instead `position_embedding_type` with "absolute" (default), "relative", <good_name_for_method_4>?

Reply:
Good point, will do.